src/libstd/panicking.rs

```diff
@@ -274,27 +269,59 @@ pub unsafe fn r#try<R, F: FnOnce() -> R>(f: F) -> Result<R, Box<dyn Any + Send>>
     //
```

Comment:
Looks like the "2." comment above is no longer accurate? The code now does `ManuallyDrop::into_inner(data.p)` in the "panicked" case, which looks very symmetric to the "didn't panic case": in both cases we move "it out of `data` and return it", it's just a different thing we move out.